static/js/containers/pages/ReceiptPage.js

```diff
@@ -98,303 +98,197 @@ export class ReceiptPage extends React.Component<Props> {
-                                {orderReceipt.receipt.card_type
-                                  ? `${orderReceipt.receipt.card_type} | `
-                                  : null}
-                                {orderReceipt.receipt.card_number
-                                  ? orderReceipt.receipt.card_number
-                                  : null}
-                              </p>
-                            </div>
-                          </div>
-                        ) : null}
-                        {orderReceipt.coupon ? (
-                          <div className="row no-gutters">
-                            <div className="col-lg-2 col-md-3 col-sm-4">
-                              <p className="label">Discount Code:</p>
-                            </div>
-                            <div className="col-lg-10 col-md-9 col-sm-8">
-                              <p className="value" id="discountCode">
-                                {orderReceipt.coupon}
-                              </p>
-                            </div>
+                        <div
+                          className="value low-line-height"
+                          id="purchaserCountry"
+                        >
+                          {countryName}
+                        </div>
+                      </dd>
+                    </dl>
+                    <dl>
+                      <dt>Email:</dt>
+                      <dd id="purchaserEmail">
+                        {orderReceipt.purchaser.email}
+                      </dd>
+                    </dl>
+                  </div>
+                  {orderReceipt.receipt ? (
+                    <div className="receipt-col">
+                      <h3>Payment Information</h3>
+                      <dl>
+                        <dt>Name:</dt>
+                        <dd>{orderReceipt.receipt.name}</dd>
+                      </dl>
+                      {orderReceipt.receipt &&
+                      orderReceipt.receipt.payment_method === "card" ? (
+                          <dl>
+                            <dt>Payment Method:</dt>
+                            <dd id="paymentMethod">
+                              {orderReceipt.receipt.card_type
+                                ? `${orderReceipt.receipt.card_type} | `
+                                : null}
+                              {orderReceipt.receipt.card_number
+                                ? orderReceipt.receipt.card_number
+                                : null}
+                            </dd>
+                          </dl>
+                        ) : orderReceipt.receipt.payment_method === "paypal" ? (
+                          <div>
+                            <dl>
+                              {orderReceipt.receipt.bill_to_email ? (
+                                <dl>
+                                  <dt>Email:</dt>
+                                  <dd>{orderReceipt.receipt.bill_to_email}</dd>
```

Comment:
Just need to add ` (Paypal account email)` after email like below
```
Email: briangro@mit.edu (Paypal account email)
Payment Method: Paypal
Discount Code: 1234567890abc (if used)
```

Reply:
It seems me just for reference to paypal in the JIRA issue for us. 
@briangrossman  can you confirm is that string really required, else I am going to merge it ?